package-lock.json

```diff
@@ -13034,7 +12860,7 @@
     },
     "yargs": {
       "version": "11.1.0",
-      "resolved": "https://registry.npmjs.org/yargs/-/yargs-11.1.0.tgz",
+      "resolved": "http://registry.npmjs.org/yargs/-/yargs-11.1.0.tgz",
```

Comment:
I'll fix these `http` things in a follow-up commit.  I have a slight understanding of why they're happening, but they're annoying.